lib/ex_unit/test/ex_unit/describe_test.exs

```diff
@@ -48,16 +48,18 @@ defmodule ExUnit.DescribeTest do
     end
 
     test "when using describe inside describe" do
-      assert_raise RuntimeError, ~r"cannot call describe/2 inside another describe", fn ->
-        defmodule Sample do
-          use ExUnit.Case
-
-          describe "hello" do
-            describe "another" do
-            end
-          end
-        end
-      end
+      assert_raise RuntimeError,
+                   ~r"cannot call ExUnit.Case.describe/2 inside another describe",
```

Comment:
Move the regex to a variable to avoid the line break.

Reply:
done